.github/scripts/check-engines-client.sh

```diff
@@ -1,46 +1,41 @@
 #!/bin/bash
 
-echo "-------------- Checking Generated Client QE Engine --------------"
+echo "-------------- Checking Generated Client QE file --------------"
 
 dir=$1
 project=$2
 
-DEFAULT_CLIENT_ENGINE_TYPE='library'
-
-# Check to see if the env var "PRISMA_CLIENT_ENGINE_TYPE" is set if not then using the default
-if [ -z "$PRISMA_CLIENT_ENGINE_TYPE" ]; then
-  echo "Using default client engine: $DEFAULT_CLIENT_ENGINE_TYPE"
-  CLIENT_ENGINE_TYPE=$DEFAULT_CLIENT_ENGINE_TYPE
-else
-  echo "Using env(PRISMA_CLIENT_ENGINE_TYPE): $PRISMA_CLIENT_ENGINE_TYPE"
-  CLIENT_ENGINE_TYPE=$PRISMA_CLIENT_ENGINE_TYPE
-fi
-
-# These are skipping for a variety of reasons like:
+# These are being skipped for a variety of reasons like:
 # - Custom project structure
 # - Custom output location
 # - They do not generate a client
+# - They do not set an PRISMA_CLIENT_ENGINE_TYPE
 # TODO Adapt tests so they also work here, or adapt project to fit into the mold
 skipped_projects=(
-  prisma-dbml-generator                   # No generated Client, so only Client stub with no engine included
-  prisma-json-schema-generator            # No generated Client, so only Client stub with no engine included
-  engine-types                            #
-  pkg                                     # No generated Client, so only Client stub with no engine included
-  aws-graviton                            # No local project at all (everything happens on server), so no `prisma` or `node_modules
-  firebase-functions                      # No local project at expected location (but in `functions` subfolder)
-  studio                                  # TODO: No generated Client in `node_modules/.prisma/client/`
-  netlify-cli                             # Client is generated into `../functions/generated/client` via use of `output`
-  jest-with-multiple-generators           # No generated Client locally in default path, both Clients have custom `output`
+  engine-types                                # No PRISMA_CLIENT_ENGINE_TYPE
```

Comment:
Random idea: it would be nice to have these as a config into the project `package.json` maybe